nmpolicy/internal/lexer/rune.go

```diff
@@ -62,3 +62,13 @@ func (l *lexer) isPlus() bool {
 func (l *lexer) isPipe() bool {
 	return l.scn.Rune() == '|'
 }
+
+func (l *lexer) matchesBooleanAt(pos int) bool {
+	if pos < 4 {
+		return rune("false"[pos]) == l.scn.Rune() || rune("true"[pos]) == l.scn.Rune()
```

Comment:
we have to check if we are at "true" or "false" to know what to match, if we don't do that something like "fruse" will be marked as boolean and will fail at parser.

Reply:
Oh of course. Good point

Reply:
Done